slides/intro-03-what-makes-a-model.qmd

```diff
@@ -285,20 +304,19 @@ $log(\frac{p}{1 - p}) = \beta_0 + \beta_1\cdot \text{distance}$
 #| fig.width: 8
 #| fig.height: 7
 
-tree_fit <- 
-  decision_tree(cost_complexity = 0.1, mode = "classification") %>%
-  fit(tip ~ distance, data = taxi_train)
+tree_fit <- decision_tree(mode = "classification") %>% 
+  #decision_tree(cost_complexity = 0.003, min_n = 50, mode = "classification") %>% 
```

Comment:
This would give us a tree with multiple splits but I tend toward the simple option with just one split as it is now. Up for discussion though!

Reply:
I'm fine with multiple splits as long as multiple means "<5" splits

Reply:
It's (very narrowly) not "< 5" 😆